src/main/java/com/google/vinet/servlets/RegistrationServlet.java

```diff
@@ -179,22 +181,71 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       return;
     }
 
-    final User user = userService.getCurrentUser();
-    final String userId = user.getUserId();
-
-    if (userId == null) {
-      response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+    final boolean registered;
+    try{
```

Comment:
spacing?

Reply:
I've fixed this now 👍 https://github.com/googleinterns/step226-2020/pull/8/commits/2ae6c631bca866f6df194ea9f018a8008343905f